nilearn/input_data/nifti_masker.py

```diff
@@ -268,8 +269,9 @@ def transform_single_imgs(self, imgs, confounds=None, copy=True):
 
         Returns
         -------
-        region_signals: 2D numpy.ndarray
-            Signal for each voxel inside the mask.
+        region_signals: 2D numpy.ndarray or MemorizedResult
```

Comment:
May be we should forget about ```MemorizedResult``` since it is not exposed to user and also it is not officially documented yet.